pkg/server/sandbox_list.go

```diff
@@ -42,16 +39,15 @@ func (c *criContainerdService) ListPodSandbox(ctx context.Context, r *runtime.Li
 	// List all sandboxes from store.
 	sandboxesInStore := c.sandboxStore.List()
 
-	resp, err := c.taskService.List(ctx, &tasks.ListTasksRequest{})
+	response, err := c.taskService.List(ctx, &tasks.ListTasksRequest{})
 	if err != nil {
 		return nil, fmt.Errorf("failed to list sandbox containers: %v", err)
 	}
-	sandboxesInContainerd := resp.Tasks
 
 	var sandboxes []*runtime.PodSandbox
 	for _, sandboxInStore := range sandboxesInStore {
-		var sandboxInContainerd *task.Task
-		for _, s := range sandboxesInContainerd {
+		var sandboxInContainerd *task.Process
```

Comment:
Hm, you are write, that's changed to `task.Process`.

And you are correct, I mistook this with client process. In the api, `Status` is actually just a field. My bad. :p